app/assets/images/payment_icons/unionpay_quickpass.svg

```diff
@@ -0,0 +1 @@
FgE0smc7yFQ5KhRGKQ4nHIdfUIzmqFnUxAbmiE3DIDqKsDEknACHIXCY/PhoIbJW8r ycGsaHvkqxPF4xV/s6KJcth5s+zgo3ymnSDA8gitSkqlPmnvdcQ40e8/EWL528Wixz7rdFCqAAAI BUlEQVRfXYapeovySJpTgbLU5GnvBvT2AT0p5A91werpgtJ1COl9e9Hdvg9CNoNAIQ/FzEIk7nXH gikTmaYM3SZONAIrFVkKXJbk/X2qtOWNns5NfczaMFQw2wqwOgtIp6aSjmvqC3BlV54oDWtUK8nR NAx9hhEQmiKKuKBRDyybKUpnNAna3KgsNgcgkzu74AemMZfBtW24bjGvnaohbVnQ4wl05fMwknWI z5gJI5lAeOZMoLEJSMa9raMqAkYAoM2nXqRu9lvD1QXfa7wUoErBq3hxeRBwtYJfkYpStrXlLA6l RpAqTh+qbfeoPqtM6EZd/V5//kRdR6sy15ZKxphS51kETvRDGWoYMJD2vNEHszAPdiDb24f+w13I 9vbAHupCUHShqzJ0iRwYLTCzANc2PdofHtguQ7YUZotyIWWyw33MaTvkOpt64G7cl8u8nbOcDgup QycSqIZ3NjWI2lGv+vFGBJN2qDVeQFO9zVqadOWslmD4nKimtWqSFFcdZhiyJhHzJsvnwRwbNrNg hEMwbQsFy4KiBUBbSEotRvn0Bl0XUiiIAoFVQIVRF4MUCkGPhiGH6Fovc4xfDGOMxOZjOB+WbwnF ClgMSjuOKFAqK5NtCSsDLaHKdo/gVaXtH/227/Xnjzf2dPAkOgwyeZy7bDiNWzY7hFw+j0w6DaTz wKFeKFkLdjYPyXKgUkA4ZVx3SRmjXI15Hg5HSY+piJLAGYFdRjxkInKS5FiiknWY0l8Q1f27u7rW 9VjW6wfMwsEumB0FxPffh84J08dXJr+1XX1SaFjlr/A11IUlNT9LMtOt9RE1ETP0hc3B6DmNSvA0 Leskw66oBF0okugKjkwd70IC48oTgZhLe3damMjuJciwWDFFhipzuhNaWfgOkH7TolMab1XUWEiL Gy6lgFW25SrNuFzNUBAHVGXl6AJWre2vhB9rrPd8rz9/vLHncukyflxE4DJSPHkhmSVQUhmDQhYT Bk63Q/G6/G9Guw/iMrNgEi2M48JxXSQTdbBMh9mWa2UcZvUbSu9hM/9WT8Fa15PP7+wazPa5qr5f MvX276FnqDLZPPZXn5SARVallYBYDwRkDc2GjLlxSWqMOnL9zEB0Ub0cWJZUtPkahJAkM5ExR5BI n2I2X2EomIczPDhkVCaqFJFrXRQ0zTUhTtviFX6YSAQHE/XEKJAaDRiVa0ijB7XiCXsEYNZmw6r4 +WUyOQ1Yx24K+WPjjArtGFngeBLcEpZDryWUh6pIrSMKcBUi92WwHcZMBmbacAu2kC5Y1q4hQdjY Zg7t6HLs7rTNOvslqU0qFA53A7mHimv6sYegyp5w7Hq7snaMdzUBl1IPqBIQ04CWsIQ58XCwOWFE 4w1yYG6dIy4yIC7QRDcqMlOUBQuaKJD6BdFlUEyJsznYtEWk7T/nF3e58Z4KcXV5vOZFQfC1qWFg oAtJdRt7q1XrhK9MwzrShjV69a1sO8hX6opPKUcPVUUc72OM8nv9+WMJvk9cOCxbJWPkfefZbgmw iPiQmFz5p8XZ7P3vFt0TXVguc3MiBjKQ3t7bN7hjSEBb2rH608w9fNDM7XWBgw6Q6gbMhzzeqmPp HVYTMpzsgDX8cuR0GgECCTrv07SI6giNM8LB2fW2OCup6MmILtepEpuhC06LLtgNqoCkIchCkE4/ XKIABgcpz2ZFAZzerVUOWCXbsiO2fz4IjA1a1QHWCLCw0mdXMZSsxlPC6to/0tBpDevoTiH/bpz7 zQeqskXFBysutcRe4ss2yTXJt8CYK7JeF2LXYGrwoOmKh7JAT06Sevf09rangH0ZsM4+FAb7gMwg kDtWwcpViPSEVY5ubx/t1o1zPzpVzKMlYuiZMFw7FJKccFgQGqIimx3T1JaYKiTiqpqIq3rUMBGT GYsKEEIinKDgsKBAoUDF0ynfbYIMkd74k57twjMGi6N4xUnLKk+MMLmGVAJ4/vvwZHtembz+xJ06 WsOqfACmAas2ZaJWwD5ixIqaPDdVuIwvtvxUXCB3HhcOI84qxjUrR2CuKwgZSxQytiym8zIGbEVM 9eWGBvqz6b4hx+07NJA7aEPeV4DbBehDNuS0BWfIwtDgyXDqV6nEviMBq/QlyXNeiUYjqiBE5Ww2 EjBNIylDDzMYQUVqbE3WNUvMblQEIS4zMawJYlQDDEUQVUGgMxNBFgRXIX5BURQlQjJBcMmrarhv ykFqlBJ2hFvCRNuyI7+rdUtUK+CMxPBXKjre9WKNcVHv9eeP7vUSzZuWSodSzBIpHHMs0XVcJlgm E+2C4DoWs00bQtZhGDBFNlQQ3P68iM49HYcPDznoTAPZApBPqWqWicagqLEBa2Bg8Hh7plcnVePX escDVvmr0dbxrEgkIplmVHXdsCWZYYFBFxmnBRRlTiIoBTRRDEkOjLp43JBcUPqegCAwRRKIEZBA bGQmUnaW0lIKO+Pv6FxA9OPjxx+AGneEqHVLWbO1olYJqk3BqT2O9EQ//wjREDmXPifnFZjNmGg5 YJYtCjlXZJmu/v6sDSmbF5y0LSBH53887YgAy7SRZyKGFBFDTh4DrwOD75St3lSBrVZxm+pzjvd1 /nvx3ysBOQgYRgBh0UFQZQhIDIYGOSDbtqpCUgVAESRJkuCIgkjeeV7xDPIjpVxHKidGG3315IBV a8dQopzp8i7sAcqK7cBxJO426uQBy5FhMgGmIyCXF5BlEnKujUxWw1AmhexDPqX9iNG8Vjg+6Tr2 /wMy/Z8+kHigBAAAAABJRU5ErkJggg==" id="image902" opacity="1" stop-opacity="1"/></svg>
```

Comment:
Can we remove unused `id` attributes if it's not used? I see some instances:

- `path834`
- `path836`

Also is the `<defs></defs>` needed?